serverless/install/install-serverless-operator.adoc

```diff
@@ -20,7 +20,7 @@ Read the following information about supported configurations and prerequisites
 * {ServerlessProductName} currently cannot be used in a multi-tenant configuration on a single cluster.
 endif::[]
 
-include::modules/serverless-cluster-sizing-req.adoc[leveloffset=+2]
+include::modules/serverless-cluster-sizing-req.adoc[leveloffset=+1]
```

Comment:
@abrennan89 Can you help me with this? I was attempting to fix an error in the build here:

```
asciidoctor: WARNING: modules/serverless-cluster-sizing-req.adoc: line 7: section title out of sequence: expected level 1, got level 2
```

But I don't think this fix is right. Because if the above conditional is true, then this won't be at the right level.

So I think there needs to be both an ifdef and an ifndef, each with the "Before you begin" heading and this include file, but then only the "Read the following..." section perhaps is what should only be in openshift-enterprise?

I'm not sure :smile: so would appreciate your help!

Reply:
Hey Andrea. I see the issue. Maybe it would help if this level 2 was included in the ifdef and I add a new ifdef for dedicated that has this as a level 1? Sorry about the build error, I didn't realize the issue at the time when I worked on this.
Let me know if you want me to put in a PR to fix it. Thanks!

Reply:
@abrennan89 Yeah something like that would work if you don't mind submitting the PR for it. Thanks!

Reply:
@bergerhoffer I think you can close this one; I put the fix in for it with our release notes that will be merged once 4.10 goes out sometime next week 🙂 https://github.com/openshift/openshift-docs/pull/43021

Reply:
Thanks @abrennan89! Closing this one